scripts/release/mule/package/deb/package.sh

```diff
@@ -7,14 +7,17 @@ echo
 date "+build_release begin PACKAGE DEB stage %Y%m%d_%H%M%S"
 echo
 
-BRANCH=${BRANCH:-$(./scripts/compute_branch.sh)}
-CHANNEL=${CHANNEL:-$(./scripts/compute_branch_channel.sh "$BRANCH")}
+if [ -z "$NETWORK" ]; then
+    echo "[$0] NETWORK=$NETWORK is missing."
+    exit 1
+fi
+
+CHANNEL=$("./scripts/release/mule/common/get_channel.sh" "$NETWORK")
 VERSION=${VERSION:-$(./scripts/compute_build_number.sh -f)}
 # A make target in Makefile.mule may pass the name as an argument.
 PACKAGE_NAME="$1"
 
-DEFAULTNETWORK=${DEFAULTNETWORK:-$(./scripts/compute_branch_network.sh "$BRANCH")}
-DEFAULT_RELEASE_NETWORK=$("./scripts/compute_branch_release_network.sh" "$DEFAULTNETWORK")
+DEFAULT_RELEASE_NETWORK=$("./scripts/compute_branch_release_network.sh" "$NETWORK")
 export DEFAULT_RELEASE_NETWORK
```

Comment:
It could.  This is all `compute_branch_release_network.sh` is doing:
```
# The main purpose of this script is to map default network for STABLE builds to MAINNET and not TESTNET

NETWORK=$1

if [ -z "${NETWORK}" ]; then
    echo "Network name required: eg 'compute_branch_release_network.sh mainnet'"
    exit -1
fi

if [ "${NETWORK}" = "testnet" ]; then
    echo "mainnet"
    exit 0
fi

echo "${NETWORK}"
```

Reply:
A script that just flips testnet to mainnet for DEFAULT_RELEASE_NETWORK is wild to me. It's required until we refactor this, though.

Reply:
It makes sense though.